Tools/ros2/jazzy_macos.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
+  ardupilot:
+    type: git
+    url: https://github.com/srmainwaring/ardupilot.git
+    version: pr/pr-dds-launch-tests
```

Comment:
The macOS `.repos` file in `ardupilot_gz` is using `master` instead. Is this specific version still required ? @srmainwaring